csrc/transformer/inference/includes/inference_context.h

```diff
@@ -97,14 +99,21 @@ class Context {
         if (!_free_memory_size) { cudaMemGetInfo(&_free_memory_size, &total_size); }
 
         size_t activation_size = 16 * hidden_dim * batch_size;
+        size_t temp_size = batch_size * num_heads * prompt_len * prompt_len * elem_size;
```

Comment:
divide the heads by mp_size here, too.

Reply:
mp_size is not available here, so I added GetAttentionUnfusedWorkspace to the Context

Reply:
@RezaYazdaniAminabadi  Also I'm first-time contributor in DeepSpeed, and it's required to approve workflow running

Reply:
fixed, all checks are passed